src/estimagic/estimation/estimate_msm.py

```diff
@@ -27,19 +27,19 @@ def estimate_msm(
     moments_cov,
     params,
     optimize_options,
+    *,
     lower_bounds=None,
     upper_bounds=None,
-    *,
     constraints=None,
     logging=False,
     log_options=None,
     simulate_moments_kwargs=None,
+    simulate_moments_and_jacobian=None,
```

Comment:
We could drop `simulate_moments_and_jacobian`. JAX can only exploit synergies for gradients, not jacobians and I don't think any other framework can.

Reply:
Okay.